pkg/services/ngalert/notifier/channels/email.go

```diff
@@ -95,6 +98,18 @@ func (en *EmailNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool,
 	return true, nil
 }
 
+func getTitleFromTemplateData(data *template.Data) string {
+	title := "[" + data.Status
+	if data.Status == string(model.AlertFiring) {
+		title += ":" + fmt.Sprintf("%d", len(data.Alerts.Firing()))
```

Comment:
This should be much more readable:
```suggestion
		title += fmt.Sprintf(":%d", %d", len(data.Alerts.Firing()))
```